cmd/vic-machine/create/create.go

```diff
@@ -424,17 +426,9 @@ func (c *Create) processParams() error {
 
 func (c *Create) processCertificates() error {
 
-	// debuglevel is a pointer now so we have to do this song and dance
-	var debug int
-	if c.Debug.Debug == nil {
-		debug = 0
-	} else {
-		debug = *c.Debug.Debug
-	}
-
 	c.certs.Networks = c.Networks
 
-	if err := c.certs.ProcessCertificates(c.DisplayName, c.Force, debug); err != nil {
+	if err := c.certs.ProcessCertificates(c.DisplayName, c.Force, *c.Debug.Debug); err != nil {
```

Comment:
I think we still need to do the check before this as c.Debug.Debug can be nil, right?